### PR TITLE
docs: move cli commands above api in nav

### PR DESCRIPTION
**Pull request by jenniferfell**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Left navigation...

Release Information >
Quick Reference >
API
CLI Commands >

Issue Number: N/A

## What is the new behavior?
Left navigation...

Release Information >
Quick Reference >
CLI Commands >
API

Given that 100% of users need the API reference at some point, given the ease of access at the bottom of the nav, and given that it doesn't have any children...this feels better. 

## Does this PR introduce a breaking change?
```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: aio/content/navigation.json [hunk 1]
```diff
@@ -612,23 +612,22 @@
...
       ]
     },
-
-    {
-      "title": "API",
-      "tooltip": "Details of the Angular packages, classes, interfaces, and other types.",
-      "url": "api"
-    },
     {
       "title": "CLI Commands",
```
Comment (Splaktar) [inline aio/content/navigation.json]: Should this section just be called “CLI”? Since it includes the “Using the CLI” sub-section.

**Review by Splaktar — APPROVED** [review]
I like the CLI coming before the API (which I see as more advanced content).

File: aio/content/navigation.json (hunk 1 above)
Comment (jenniferfell) [reply aio/content/navigation.json]: Changed "Using" to "Overview" to be more clear that this first entry isn't a guide to all the things you can do with the CLI; it's an introduction/overview to the CLI tool, commands, and syntax. "Using" is described throughout the docs, in the related guides such as Getting Started (using ng new), i18n, universal, routing, building, deployment, etc.

**Comment by jenniferfell** [comment]
Changes: 
* Moved CLI above API in left nav (user vote was 6:1 in favor of this move)
* Renamed "Using the CLI" to "Overview" because this isn't a guide to all things you can do with the CLI. It's an overview/intro. 

@IgorMinar Ready for your review, with requests or approval. Thanks!

**Review by IgorMinar — APPROVED** [review]
ok. I admit. this is better :-)

**Comment by IgorMinar** [comment]
caretaker note: ci is flaky

**Pull request closed without merging**